CHANGELOG.md

```diff
@@ -1,6 +1,31 @@
 <!-- markdownlint-disable no-duplicate-heading -->
 # Change Log
 
+## 0.7.0
+
+### Removals
+
+* Dropped the option to configure ansible-vault path (#318) @ssbarnea
+* Dropped the option to configure ansible-playbook location (#317) @ssbarnea
+
+### Minor Changes
+
+* Upgraded the `@ansible/ansible-language-server` dependency to 0.3.0 (#333) @ssbarnea
+  * Added support for nested module options (suboptions) @tomaciazek
+  * Updated container cleanup logic for execution environment @ganeshrn
+* Switched the default execution environment to `ansible/creator-ee:latest`
+  (#331) @ganeshrn @ssbarnea
+* Enabled auto-selecttion of the only vault-id (#298) @jeinwag
+
+### Bugfixes
+
+* Corrected some unspecified configurable settings (#307) @ssbarnea
```

Comment:
@ssbarnea I was hoping you'd add a bit of clarity here.

Reply:
I just added you version of the text. Lets see how many people complain about it. Considering that 99% of App Store release notes have a constant single line sounding just like "We made it better!", we already did far more than most.

Reply:
That's a pretty low bar. You may have noticed that I prefer high quality release notes. Most people don't know that they can complain about bad release notes but it's not an excuse to maintain low quality.

[![](https://imgs.xkcd.com/comics/update_notes.png)](https://xkcd.com/2010/)